misc/services/mobile-webui/mobile-webui-frontend/src/apps/huManager/containers/HUManagerScreen.jsx

```diff
@@ -33,24 +42,24 @@ const HUManagerScreen = () => {
   const [currentLocatorQRCode, setCurrentLocatorQRCode] = useState();
   const [handlingUnitInfo, setHandlingUnitInfo] = useHandlingUnitInfo();
 
-  const resolveScannedBarcode = ({ scannedBarcode }) => {
-    return api.getHUByQRCode(scannedBarcode).then((handlingUnitInfo) => ({ handlingUnitInfo }));
-  };
-
-  const onResolvedResult = (result) => {
-    //console.log('onResolvedResult', { result });
-    const { handlingUnitInfo } = result;
-    setHandlingUnitInfo(handlingUnitInfo);
-  };
+  const { url } = useRouteMatch();
+  useEffect(() => {
```

Comment:
why adding this back. isn't `useScreenDefinition` already doing this?

Reply:
it got mixed up in the merging, I'll take it down